cmd/serve.go

```diff
@@ -119,6 +131,8 @@ func init() {
 	serveCmd.Flags().BoolVarP(&VerboseFlag, "verbose", "v", false, "show log messages")
 	serveCmd.Flags().BoolVarP(&BenchmarkFlag, "benchmark", "b", false, "display build time statistics")
 	serveCmd.Flags().BoolVarP(&SSLFlag, "ssl", "s", false, "ssl/tls encryption to serve localhost over https")
+	serveCmd.Flags().BoolVarP(&build.Doreload, "hot-reload", "H", false, "Enable hot reload")
```

Comment:
I've been using the terms "hot reload" and "live reload" interchangeable, zaydek had a [good point](https://github.com/plentico/plenti/issues/99#issuecomment-773995500) though:

> HMR as in when state is persisted between refreshes, live reload as in hard reloads

Do you think we should change this to `--live-reload` / `-L` to prevent confusion with HMR?